generators/bootstrap-application-base/generator.ts

```diff
@@ -156,6 +156,7 @@ export default class BootstrapApplicationBase extends BaseApplicationGenerator {
             }
             return 'dist/';
           },
+          buildToolUnknown: ({ buildTool }) => !['gradle', 'maven'].includes(buildTool!),
```

Comment:
Shouldn't it be in server?

Reply:
It's used in client generators. It's possible to add an undefined value, but decided to put the correct value.